src/main/Database/users.csv

```diff
@@ -0,0 +1,5 @@
+name,password,createTime,friendList
```

Comment:
the friends list should be stored in a profile.csv so you'll be using different headers